src/main/java/com/betadevels/onlineshopping/resources/SubscriptionResource.java

```diff
@@ -54,4 +65,28 @@ public Response getAllSubscriptions(@Context HttpHeaders headers, @Context HttpS
 		return Response.status( Response.Status.OK ).entity( subscriptionListResponse ).build();
 	}
 
+
+	@POST
+	@Path("/skip/{subscription_id}")
+	@UnitOfWork
+	@Timed
+	public Response skipDueDate(@Context HttpHeaders headers, @Context HttpServletRequest request, @NotNull @PathParam("subscription_id") Long subscriptionId)
+	{
+		SubscriptionResponse subscriptionResponse=skipDueDateAction.withId(subscriptionId).invoke();
+		return Response.status(Response.Status.OK).entity(subscriptionResponse).build();
+	}
+
+	@POST
```

Comment:
Updates should be of type PUT and hence they wont be needing the word 'update' in their path.